### DISCUSSION TITLE
Other Libraries in Consideration

### DISCUSSION THREAD

**Discussion by BradKML** [boyter/scc#296] [Ideas]
There is a Cyclomatic Complexity Analyzer that covers 18 languages (incl TypeScript, Swift, Scala, GDScript, Golang, Lua, Rust and Kotlin) https://github.com/terryyin/lizard

It would be wise to compare the token used between the repos and see it anything is missing?

**Comment by boyter** [comment]
Sounds like a reasonable thing to do. Ill have a look at it when I get back to `scc` sometime. Been distracted with other projects recently.

**Reply by BradKML** [reply]
Don't worry, take your time.
